Missing libisl when installing gcc in spack

**Description**
Following the [`spack` install instructions](https://picongpu.readthedocs.io/en/latest/install/instructions/spack.html) for `PIConGPU 0.4.0`, an error occurs when installing `gcc` via
```
spack install gcc@5.4.0
```

**Error log**
```
==> Installing gcc
==> Fetching http://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.bz2
[ ... ]
==> Error: ProcessError: Command exited with status 2:
    'make' '-j8'

6 errors found in build log:
[ ... ]
  >> 15250 /tmp/berceanu/spack-stage/spack-stage-e9ZEMa/gcc-5.4.0/spack-build/./gcc/cc1: error while loading shared libraries: libisl.so.13: cannot open shared object file: No such file or directory
[ ... ]
  >> 15370 checking for suffix of object files... configure: error: in `/tmp/berceanu/spack-stage/spack-stage-e9ZEMa/gcc-5.4.0/spack-build/x86_64-unknown-linux-gnu/libgcc':
  >> 15371 configure: error: cannot compute suffix of object files: cannot compile
     15372 See `config.log' for more details.
     15373 Makefile:16271: recipe for target 'configure-stage2-target-libgcc' failed
  >> 15374 make[2]: *** [configure-stage2-target-libgcc] Error 1
     15375 make[2]: Leaving directory '/tmp/berceanu/spack-stage/spack-stage-e9ZEMa/gcc-5.4.0/spack-build'
     15376 Makefile:20856: recipe for target 'stage2-bubble' failed
  >> 15377 make[1]: *** [stage2-bubble] Error 2
     15378 make[1]: Leaving directory '/tmp/berceanu/spack-stage/spack-stage-e9ZEMa/gcc-5.4.0/spack-build'
     15379 Makefile:902: recipe for target 'all' failed
  >> 15380 make: *** [all] Error 2
```
**System info**
```
$ uname -m && cat /etc/*release
x86_64
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.3 LTS"
NAME="Ubuntu"
VERSION="16.04.3 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.3 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial
```
```
$ gcc --version
gcc (Ubuntu 5.4.0-6ubuntu1~16.04.5) 5.4.0 20160609
```
```
$ uname -a
Linux berceanu-GL552JX 4.13.0-26-generic #29~16.04.2-Ubuntu SMP Tue Jan 9 22:00:44 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

--- a/docs/source/install/instructions/spack.rst
+++ b/docs/source/install/instructions/spack.rst
@@ -26,9 +26,7 @@ First `install spack <http://spack.readthedocs.io/en/latest/getting_started.html
    spack bootstrap
 
    # install a supported compiler
-   spack install gcc@5.4.0
-   spack load gcc@5.4.0
-   spack compiler add
+   spack compiler list | grep gcc@5.4.0 || spack install gcc@5.4.0 && spack load gcc@5.4.0 && spack compiler add
 
    # add the PIConGPU repository
    git clone https://github.com/ComputationalRadiationPhysics/spack-repo.git $HOME/src/spack-repo

